yb-voyager/src/query/queryissue/constants.go

```diff
@@ -457,8 +457,8 @@ Note: If the table is created as colocated, this hotspot concern can safely be i
 
 // Object types
 const (
-	CONSTRAINT_NAME           = "ConstraintName"
-	FUNCTION_NAMES            = "FunctionNames"
+	CONSTRAINT_NAME           = "constraint_name"
+	FUNCTION_NAMES            = "function_names"
```

Comment:
Confused why some of the const vars are upper case vs the new ones in lower case?